scripts/spack/packages/serac/package.py

```diff
@@ -115,17 +115,17 @@ class Serac(CachedCMakePackage, CudaPackage):
     # Note: Certain combinations of CMake and Conduit do not like +mpi
     #  and cause FindHDF5.cmake to fail and only return mpi information
     #  (includes, libs, etc) instead of hdf5 info
-    depends_on("hdf5@1.8.21+hl~mpi~shared")
+    depends_on("hdf5@1.8.21+hl~mpi~shared build_type=Release")
 
     depends_on("camp")
 
     depends_on("raja~shared~examples~exercises", when="+raja")
     depends_on("raja~openmp", when="+raja~openmp")
     depends_on("raja+openmp", when="+raja+openmp")
 
-    depends_on("umpire@2022.03.1~shared~examples~device_alloc", when="+umpire")
-    depends_on("umpire~openmp", when="+umpire~openmp")
-    depends_on("umpire+openmp", when="+umpire+openmp")
+    depends_on("umpire@2022.03.1~shared~examples~device_alloc build_type=Release", when="+umpire")
+    depends_on("umpire~openmp build_type=Release", when="+umpire~openmp")
+    depends_on("umpire+openmp build_type=Release", when="+umpire+openmp")
```

Comment:
```suggestion
    depends_on("umpire~openmp", when="+umpire~openmp")
    depends_on("umpire+openmp", when="+umpire+openmp")
```

These do not need to be on every line. They are cumulative.